### PR TITLE
created home route

### PR DESCRIPTION
**Pull request by Wyrd00**
relates #8
hurry up mario

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +18,15 @@
...
     "express": "^4.15.4",
     "express-handlebars": "^3.0.0",
     "istanbul": "^0.4.5",
+    "pg": "^7.1.2",
     "serve-favicon": "^2.4.3",
     "tap-spec": "^4.1.1",
-    "tape": "^4.8.0",
-    "env2": "^2.2.0",
-    "pg": "^7.1.2"
+    "tape": "^4.8.0"
```
Comment (matthewdking) [inline package.json]: tape, tap-spec and istanbul should be dev dependencies

**Review by matthewdking — CHANGES_REQUESTED** [review]

**Review by matthewdking — APPROVED** [review]
:tada:

**Review by sajedazoabi — APPROVED** [review]

File: src/controllers/home.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = (req, res) => {
+	res.render('home', { activePage: {home: true}});
```
Comment (Karyum) [inline src/controllers/home.js:2]: copy pasting workshop much? :stuck_out_tongue_winking_eye:

**Comment by matthewdking** [comment]
@Karyum :hourglass_flowing_sand:

File: src/views/error.hbs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+	{{>htmlHead}}
+<body>
+
+	<main class="main">
```
Comment (Karyum) [inline src/views/error.hbs]: indentation is wierd here

File: src/views/home.hbs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<section class="main__container--limited-width">
+	<h1 class="main__title">HOME</h1>
```
Comment (Karyum) [inline src/views/home.hbs]: and here

File: src/views/layouts/main.hbs [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<html>
+	{{>htmlHead}}
+	<body>
+
+	<main class='main'>
+
+		{{{body}}}
```
Comment (Karyum) [inline src/views/layouts/main.hbs]: and here

**Review by Karyum — COMMENTED** [review]
good job you guys just a really smal change :+1:

**Comment by Karyum** [comment]
@matthewdking :hourglass_flowing_sand:

**Comment by matthewdking** [comment]
@Karyum :hourglass: :hourglass: :hourglass:

File: src/controllers/home.js (hunk 2 above)
Comment (Wyrd00) [reply src/controllers/home.js:2]: gasp slander!

**Review by Karyum — APPROVED** [review]
sneaked in couple of images i saw that :eyes:

**Pull request merged**